src/elements/content-sidebar/MetadataSidebar.js

```diff
@@ -405,6 +405,8 @@ class MetadataSidebar extends React.PureComponent<Props, State> {
                 }
                 className="bcs-metadata"
                 title={SidebarUtils.getTitleForView(SIDEBAR_VIEW_METADATA)}
+                aria-labelledby="sidebarmetadata"
+                id="sidebarmetadata-content"
```

Comment:
You will have to use lodash/uniqueid to generate one, or pass it down from ContentSidebar.js which generates an ID and use that in concatenation or something